core/domain/question_jobs_one_off_test.py

```diff
@@ -123,12 +123,12 @@ def test_migration_job_converts_old_question(self):
         correctly and an old question is converted to new
         version.
         """
-        # Generate question with old(v27) state data.
-        self.save_new_question_with_state_data_schema_v27(
+        # Generate question with old(v31) state data.
```

Comment:
Not sure you should be changing this? Please read the context of the function before you change things.